spec-tests/src/test/java/org/candlepin/spec/content/ConditionalContentSpecTest.java

```diff
@@ -725,4 +725,7 @@ private String getCertFromEnt(JsonNode ent) {
         return certs.get(0).get("cert").asText();
     }
 
+    private String randomContentId(int length) {
+        return StringUtil.random(5, StringUtil.CHARSET_NUMERIC).replace('0', '1');
```

Comment:
Two issues here:
1) same as above where we're erroneously replacing after generating rather than just generating as intended
2) the length argument is ignored entirely